src/DssSpell.sol

```diff
@@ -45,67 +50,90 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE      = ;
 
-    uint256 internal constant FOUR_NINE_PCT_RATE = 1000000001516911765932351183;
+    address internal constant D3M_HUB                   = 0x79Dcb858D6af6FeD7A5AC9B189ea14bC94076dfb;
+    address internal constant D3M_MOM                   = 0x8aBafFe006205e306F4307EE7b839846CD1ff471;
 
-    uint256 internal constant RAD = 10 ** 45;
-
-    VatLike internal immutable vat = VatLike(DssExecLib.vat());
-
-    function actions() public override {
-
-        uint256 lineReduction;
-        uint256 line;
-
-        // ---------- RWA008-A Offboarding ----------
-        // Poll: N/A
-        // Forum: https://forum.makerdao.com/t/security-tokens-refinancing-mip6-application-for-ofh-tokens/10605/51
-
-        // Set RWA008-A Debt Ceiling to 0
-        (,,,line,) = vat.ilks("RWA008-A");
-        lineReduction += line;
-        DssExecLib.setIlkDebtCeiling("RWA008-A", 0);
-
-        // ---------- First Stage of Offboarding ----------
-        // Poll: https://vote.makerdao.com/polling/QmPwHhLT
-        // Forum: https://forum.makerdao.com/t/decentralized-collateral-scope-parameter-changes-1-april-2023/20302
+    address internal constant SPARK_D3M_PLAN            = 0x1fB2cF94D896bB50A17dD1Abd901172F088dF309;
+    address internal constant SPARK_D3M_POOL            = 0x8b6Ae79852bcae012CBc2244e4ef85c61BAeCE35;
+    address internal constant SPARK_D3M_ORACLE          = 0xa07C4eDB18E4B3cfB9B94D9CD348BbF6d5a7f4c2;
 
-        // Set YFI-A line to 0
-        (,,,line,) = vat.ilks("YFI-A");
-        lineReduction += line;
-        DssExecLib.removeIlkFromAutoLine("YFI-A");
-        DssExecLib.setIlkDebtCeiling("YFI-A", 0);
+    address internal constant SPARK_ADAI                = 0x4480b29AB7a1b0761e6d0d480325de28B7266d73;
+    address internal constant SPARK_DAI_STABLE_DEBT     = 0xD72630D78157E1a2feD7A329873Bfd496704403D;
+    address internal constant SPARK_DAI_VARIABLE_DEBT   = 0xa99d874d26BdfD94d474Aa04f4f7861DCD55Cbf4;
+    address internal constant SPARK_POOL_CONFIGURATOR   = 0xe0C7ec61cC47e7c02b9B24F03f75C7BC406CCA98;
 
-        // Set MATIC-A line to 0
-        (,,,line,) = vat.ilks("MATIC-A");
-        lineReduction += line;
-        DssExecLib.removeIlkFromAutoLine("MATIC-A");
-        DssExecLib.setIlkDebtCeiling("MATIC-A", 0);
+    address internal constant WBTC                      = 0x91277b74a9d1Cc30fA0ff4927C287fe55E307D78;  // Please note this is not the same WBTC as in Maker
+    address internal constant INTEREST_RATE_STRATEGY    = 0x491acea4126E48e9A354b64869AE16b2f27BE333;
```

Comment:
Checking Addresses:

- [x] `D3M_HUB` -> new `D3MHub` instance
- [x] `D3M_MOM` -> new `D3MMom` instance
- [x] `SPARK_D3M_PLAN` -> new `D3MAaveTypeBufferPlan` instance
- [x] `SPARK_D3M_POOL` -> new `D3MAaveV3NoSupplyCapTypePool` instance
- [x] `SPARK_D3M_ORACLE` new `D3MOracle` instance
- [x] `SPARK_ADAI` -> Upgradeable Proxy (see below)
- [x] `SPARK_DAI_STABLE_DEBT` -> Upgradeable Proxy (see below)
- [x] `SPARK_DAI_VARIABLE_DEBT` -> Upgradeable Proxy (see below)
- [x] `SPARK_POOL_CONFIGURATOR` -> Upgradeable Proxy (see below)
- [x] `WBTC` -> new `MintableERC20` instance - anyone can mint
- [x] `INTEREST_RATE_STRATEGY` -> new `DaiInterestRateStrategy` instance


Immutable Upgradeable Proxy instances:

Description             | Proxy Address                                | Owner                                        | Obs.
-----------             | -------------                                | -----                                        | ----
SPARK_D3M_PLAN          | `0x1fB2cF94D896bB50A17dD1Abd901172F088dF309` | `0xC911B590248d127aD18546B186cC6B324e99F02c` | `PoolAddressProvider` owned by `0xfA0e305E0f46AB04f00ae6b5f4560d61a2183E00` (EOA)
SPARK_DAI_STABLE_DEBT   | `0xD72630D78157E1a2feD7A329873Bfd496704403D` | `0xC911B590248d127aD18546B186cC6B324e99F02c` | `PoolAddressProvider` owned by `0xfA0e305E0f46AB04f00ae6b5f4560d61a2183E00` (EOA)
SPARK_DAI_VARIABLE_DEBT | `0xa99d874d26BdfD94d474Aa04f4f7861DCD55Cbf4` | `0xC911B590248d127aD18546B186cC6B324e99F02c` | `PoolAddressProvider` owned by `0xfA0e305E0f46AB04f00ae6b5f4560d61a2183E00` (EOA)
SPARK_POOL_CONFIGURATOR | `0xe0C7ec61cC47e7c02b9B24F03f75C7BC406CCA98` | `0x026a5B6114431d8F3eF2fA0E1B2EDdDccA9c540E` | `PoolAddressProvider` owned by `0x5DCdbD3cCF9B09EAAD03bc5f50fA2B3d3ACA0121` (MCD_PAUSE_PROXY :white_check_mark:)

I have 2 questions: 

1. Who is `0xfA0e305E0f46AB04f00ae6b5f4560d61a2183E00`? 
2. Should they have rights to upgrade `SPARK_D3M_PLAN`, `SPARK_DAI_STABLE_DEBT` and `SPARK_DAI_VARIABLE_DEBT` through the `PoolAddressProvider` they control?

Reply:
Got it working:

```
[⠒] Compiling...
[⠑] Compiling 1 files with 0.8.19
[⠘] Solc 0.8.19 finished in 466.62ms
Compiler run successful

Running 1 test for test/ProxyOwnerChecker.t.sol:ProxyOwnerCheckerTest
[PASS] testProxyAdmin() (gas: 34215)
Test result: ok. 1 passed; 0 failed; finished in 2.52s
```